src/main/java/Task.java

```diff
@@ -0,0 +1,66 @@
+import java.text.ParseException;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected String type;
+    protected String deadline;
+    protected String event;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+    }
+
+    public String getType() {
+        /*
+        if (type.equals("todo")) {
+            return "T";
+        } else if (type.equals("deadline")) {
+            return "D";
+        } else if (type.equals("event")){
+            return "E";
+        }
+        */
+        return "error";
+    }
+
+    public String getDesc() {
+        return this.description;
+    }
+
+    public String getAt() {
```

Comment:
Is this an incomplete method implementation? Maybe you could leave unfinished code in a separate in-development branch instead?